package.json

```diff
@@ -40,7 +39,6 @@
   "scripts": {
     "doc": "documentation build --github --format md --output docs/api.md cli.js lib/**",
     "test": "yarn lint && tape test/*.test.js",
-    "coverage": "TESTING=true nyc tape 'test/**/*.js' && nyc report --reporter=text-lcov > coverage.lcov && codecov",
```

Comment:
Ran into this on another repo.   It is OK to still leave the calls to nyc tape, just the  && codecov needs to be removed